samples/wifi/radio_test/sample_description.rst

```diff
@@ -145,6 +145,33 @@ Testing
               wifi_radio_test tx 1
 
 
+         * To run a continuous Orthogonal frequency-division multiplexing (OFDM) TX traffic sequence with the following configuration:
+
+           * Channel: 11
+           * Frame duration: 2708 µs
+           * Inter-frame gap: 4200 µs
+           * Edge backoff: 3 dB
+           * Antenna gain: 2 dB
+
+           Execute the following sequence of commands:
+
+           .. code-block:: console
+
+              wifi_radio_test init 11
+              wifi_radio_test tx_pkt_rate 12
+              wifi_radio_test tx_pkt_len 4000
+              wifi_radio_test tx_power 4
```

Comment:
Increase the example tx_power. Otherwise, we see 4 - 2 -3 = -1dBm power. This will be lower than lower tx power limit.